protocol/errors.json

```
@@ -198,6 +198,8 @@
   "92000": "invalid object message",
   "92001": "objects limit exceeded",
   "92002": "unable to submit operation on tombstone object",
+  "92003": "unable to fetch state tree with tombstone object as root",
+  "92004": "unable to fetch state tree for not found object as root",
```

```suggestion
  "92004": "object not found",
```

per suggestion in https://github.com/ably/realtime/pull/7212#discussion_r1993943642